itest/universe_pagination_test.go

```diff
@@ -22,15 +22,15 @@ const (
 
 	// testPageSizeSmall is the page size to use when fetching data from the
 	// universe rpc. We use a small page size to test pagination.
-	testPageSizeSmall = 2
+	testPageSizeSmall = 10
 
 	// testGroupSize is the size of the asset group we mint in the
 	// testUniversePaginationSimple test.
 	testGroupSize = 79
 )
 
 func testUniversePaginationSimple(t *harnessTest) {
-	mintSize := 255
+	mintSize := 50
```

Comment:
was the issue here that we had 255 mints and a page size of 2, resulting in 100++ pages (and calls)?

Reply:
Correct. Reducing this took more than 10 seconds off the itest.